src/helpers/_typography.scss

```
@@ -171,7 +172,18 @@
 
 @mixin govuk-font($size, $weight: regular, $tabular: false, $line-height: false) {
   @if $tabular {
-    @include govuk-typography-common($font-family: $govuk-font-family-tabular);
+    // if govuk-font-family-tabular is set use $govuk-font-family-tabular
+    @if $govuk-font-family-tabular != false {
```

Non-blocking, but I think this can just be

```suggestion
    @if $govuk-font-family-tabular {
```